packages/terra-date-picker/tests/wdio/date-picker-spec.js

```diff
@@ -328,4 +328,22 @@ describe('Date Picker', () => {
     Terra.should.matchScreenshot();
     Terra.should.beAccessible({ rules: ignoredA11y });
   });
+
+  describe('Key Limitations', () => {
+    before(() => {
+      browser.url('/#/raw/tests/terra-date-picker/date-picker/date-picker-default');
+      browser.execute(() => {
+        // Removes the blinking cursor to prevent screenshot mismatches.
+        document.querySelector('input[name="terra-date-date-input"]').style.caretColor = 'transparent';
+      });
+    });
+
+    it('sets the date', () => {
+      browser.setValue('input[name="terra-date-date-input"]', '0');
+      browser.keys('a1.b2/;3');
+    });
+
+    Terra.should.matchScreenshot();
+    Terra.should.beAccessible({ rules: ignoredA11y });
```

Comment:
want to use `Terra.should.validateElement` instead of these two functions?

Reply:
should be fixed here https://github.com/cerner/terra-framework/pull/595/commits/abba523319b419b80af7a36f29c869ca441d6e8f, but I noticed most of the rest tests in this test use the older way of doing tests.

Reply:
Yeah, eventually we should refactor tests to use that function. I'm not opposed to you doing that for this file though :D